src/cli/build.helpers.ts

```diff
@@ -209,3 +209,24 @@ export async function generateYamlAndWriteToDisk(obj: {
   await assets.generateHelmChart(webhookConfigGenerator, outputDir);
   console.info(`✅ K8s resource for the module saved to ${yamlPath}`);
 }
+
+export interface BuildOpts {
```

Comment:
Mmm, the name "BuildOpts" seems too general to me since this only covers 3 of the possible options (and not all of the "build opts").  Would be great if the name for this interface reflected the scaled-down nature of it's usage, something like... I dunno, "ImageBuildOpts", "ImabeBuildDeconflict", or "ImageSpecOpts" maybe.  Just something that doesn't suggest that it's more than it is.